tidb-cloud/database-schema-concepts.md

```diff
@@ -119,12 +119,6 @@ In TiDB, you can either [add a secondary index to an existing table](/develop/de
 
 ### Vector index
 
-For the following TiDB deployment options, TiDB supports vector data types and vector search indexes.
-
-- TiDB Cloud Serverless
-
-- TiDB Self-Managed v8.4.0 or later versions
-
 In TiDB, a vector index is a specialized index designed for efficient approximate nearest neighbor (ANN) searches over columns containing vector data. Vector indexes, particularly the HNSW (Hierarchical Navigable Small World) algorithm, allow K-nearest neighbors (KNN) searches to identify the closest data points in a vector space quickly. This significantly speeds up query performance, enabling results in milliseconds compared to brute-force methods.
```

Comment:
Shoulde we mention Vector index is still in beta? @qiancai

Reply:
@joey-yez Yes, I've added that in [`00bb33e` (#21368)](https://github.com/pingcap/docs/pull/21368/commits/00bb33e1f80565957c8641d6e4c56e445ba2b83a).